ores/applications/util.py

```diff
@@ -30,25 +33,40 @@ def build_config(config_dirs=DEFAULT_DIRS):
     return config
 
 
-def configure_logging(verbose, debug, logging_config=None):
-    # Load logging config if specified
+def configure_logging(verbose=False, debug=False, logging_config=None, **kwargs):
+    # Load logging config if specified.  If no config file is specified, we
+    # make a half-hearted attempt to find a distributed logging_config.yaml
+    # in the current working directory.
+    if logging_config is None:
+        if os.path.exists(DEFAULT_LOGGING_CONFIG):
+            logging_config = DEFAULT_LOGGING_CONFIG
+
     if logging_config is not None:
         with open(logging_config) as f:
             logging_config = yamlconf.load(f)
             logging.config.dictConfig(logging_config)
 
+        # Secret sauce: if running from the console, mirror logs there.
+        if sys.stdin.isatty():
+            handler = logging.StreamHandler(stream=sys.stdout)
```

Comment:
sys.stderr might be better

Reply:
Good idea—I also confirmed that this is the default behavior of logging.basicConfig, see https://github.com/python/cpython/blob/master/Lib/logging/__init__.py#L956